golden-tests/run-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Let's add a section about these tests to README with all prerequisites needed for running (e.g. that `nix-diff` should be in `PATH`)